### DISCUSSION TITLE
Cloudrun deployment issue

### DISCUSSION THREAD

**Discussion by CVxTz** [zauberzeug/nicegui#2891] [Q&A] [answered]
### Question

Hello! 
I am having an issue with cloudrun deployment where the page refreshes repeatedly but only when I define max-instances > 1.
With max-instances==1, it all works fine.

I even tried with --session-affinity parameter when deploying (It attempts to route the users to the same node) and it did not make much of a difference.

Do you have any advice as to where I should start to solve this issue ?

I am using nicegui==1.4.21 and using the same setup as the FastAPI example that you have in the examples folder.

[Screencast from 14-04-2024 16:11:51.webm](https://github.com/zauberzeug/nicegui/assets/13545260/9ba504c5-44d3-4ce9-8561-eb3c8a12354d)

**Comment by CVxTz** [comment, marked as answer]
I am going to answer myself here in case someone else has the same issue :laughing: 
The key is to add the option --no-cpu-throttling too along with --session-affinity. This way the CloudRun keeps the CPU alive between requests and does not kill the backend to just restart it again a few hundred milliseconds later. This seems to solve the problem.

**Reply by network-dave** [reply]
This is extremely appreciated, thank you so much !!!
